Cargo.toml

```diff
@@ -24,9 +24,11 @@ codec = { package = "parity-scale-codec", version = "3", default-features = fals
 	"bit-vec",
 ] }
 scale-info = { version = "2.5.0", features = ["bit-vec"] }
-avail-core = {  git = "https://github.com/availproject/avail-core", tag = "core-node-7", default-features = false, features = ["serde", "runtime"] }
+avail-core = {  git = "https://github.com/availproject/avail-core", default-features = false, branch="mmp-integration", features = ["serde", "runtime"] }
+kate = {  git = "https://github.com/availproject/avail-core", default-features = false, branch="mmp-integration", features=["serde"]}
 hex = { version = "0.4" }
 base58 = { version = "0.2.0" }
+poly-multiproof = { git = "https://github.com/availproject/poly-multiproof", ref="494468b86ac8e16dca21d465d921990c578eaad3", default-features = false, features = ['ark-bls12-381', "blst"]}
```

Comment:
This is not necessary. We already have access to this lib via kate. `use kate::pmp::`